### ISSUE TITLE
Chocolatey Package for OGCS alpha releases

### ISSUE THREAD

**Issue by nathanspectacular** [phw198/OutlookGoogleCalendarSync#1271]
**Is your feature request related to a problem? Please describe.**
I do not want to think about upgrading software.  I want software to automatically upgrade.  Yet, have the power of reverting to a specific version in case the latest version has a significant bug.

Furthermore, when I get a new laptop or have to reinstall the OS, I have a script that installs and configures all of the software I want.  Instead of spending days installing and configuring software, the script runs for about 2 hours.  Everything is ready to go.

I use Chocolately package manager to install all of the software.  This downloads the latest version and installs it.  Each week, I run the Chocolately update all command to upgrade all of the software.

**Describe the solution you'd like**
Please create a Chocolately package and make it available at https://community.chocolatey.org/.  This will allow me to automate installation and upgrades.

**Describe alternatives you've considered**
None

**Comment by phw198** [comment]
You mean like on #520? :slightly_smiling_face: 

~~Admittedly, I think that does need updating to perform update to v2.9.0 beta...~~

When you say "the latest" are you wanting alpha releases, or just betas?

**Comment by nathanspectacular** [comment]
I mean betas or in the future better than beta (official?).  With Chocolately, you can setup several packages (i.e. one package for alpha, one package for beta and one package for official).  The person can then install the package for the release channel they want and get the latest version for that channel.

**Comment by phw198** [comment]
Yes, that's why I asked as #520 already covers beta releases. If there's enough interest, I'm happy to spin up alpha chocolatey release package as well - please upvote here.
